samples/navigate-route-with-rerouting/src/main/java/com/esri/arcgismaps/sample/navigateroutewithrerouting/screens/NavigateRouteWithReroutingScreen.kt

```diff
@@ -126,20 +129,44 @@ fun NavigationRouteInfo(
             .animateContentSize(),
         verticalArrangement = Arrangement.spacedBy(8.dp)
     ) {
-        Text(
-            text = distanceRemainingText,
-            style = MaterialTheme.typography.bodyMedium
-        )
+        Row {
+            Text(
+                text = stringResource(R.string.distance_remaining),
+                style = MaterialTheme.typography.bodyMedium,
+                fontWeight = FontWeight.Bold
+            )
+            Text(
+                text = distanceRemainingText,
+                style = MaterialTheme.typography.labelLarge,
+                fontWeight = FontWeight.Light
+            )
+        }
 
-        Text(
-            text = timeRemainingText,
-            style = MaterialTheme.typography.bodyMedium
-        )
+        Row {
+            Text(
+                text = stringResource(R.string.time_remaining),
+                style = MaterialTheme.typography.bodyMedium,
+                fontWeight = FontWeight.Bold
+            )
+            Text(
+                text = timeRemainingText,
```

Comment:
To improve visibility, can you add a space in between the title and text?
applies to the others as well